src/mlpack/core/tree/binary_space_tree/typedef.hpp

```diff
@@ -135,6 +135,20 @@ using MeanSplitBallTree = BinarySpaceTree<MetricType,
                                           bound::BallBound,
                                           MeanSplit>;
 
+template<typename MetricType, typename StatisticType, typename MatType>
+using RPTreeMax = BinarySpaceTree<MetricType,
+                                  StatisticType,
+                                  MatType,
+                                  bound::HRectBound,
+                                  RPTreeMaxSplit>;
+
+template<typename MetricType, typename StatisticType, typename MatType>
+using RPTreeMean = BinarySpaceTree<MetricType,
+                                  StatisticType,
+                                  MatType,
+                                  bound::HRectBound,
+                                  RPTreeMeanSplit>;
```

Comment:
I think for these, `MaxSplitRPTree` and `MeanSplitRPTree` might be more natural names.  Which do you think is the "default"?  If the max split is the more common one, then I would just call that `RPTree`.  This is kind of what I've done with the other types of trees---only specify the split type if it's not the "usual" split type.